layouts/_default/baseof.html

```diff
@@ -1,5 +1,5 @@
-<!DOCTYPE html>
-<html>
+<!DOCTYPE html >
+<html lang="{{ with .Site.LanguageCode }}{{ . }}{{ else }}en-US{{ end }}" >
```

Comment:
Are you sure that `with` will work with `else`? I think you'll need to use `if` here.

docs: https://gohugo.io/functions/with/

Reply:
from my testing it worked. weird they didnt unclude with ... else ... on the function's page.
the pattern is in the examples though.

https://gohugo.io/templates/introduction/#example-2-with--else